src/appshell/view/aboutmodel.cpp

```diff
@@ -36,7 +38,7 @@ AboutModel::AboutModel(QObject* parent)
 
 QString AboutModel::museScoreVersion() const
 {
-    return QString::fromStdString(configuration()->museScoreVersion());
+    return QString::fromStdString(configuration()->museScoreVersion()) + (mu::framework::Version::unstable() ? " development build" : "");
```

Comment:
We normally try to make a distinction between "development build" (PR build) and "nightly build" (master branch). Is it possible to access the BUILD_CONFIG string here? If not then I think we should just say "unstable prerelease" or something like that here.

Reply:
Well, we don't, never did as far as I can tell, in 3.x we used "Unstable prerelease of  3..." if and when `Version::unstable()` returned `true` . We can use that wording for master too of course, but that's not really what you originally requested?

Reply:
"development build" is the new name for "nighly build" (and since quite long), so the same thing really. 
I'm noit sure whether we can distinguish a PR artifact from a build of a merged PR.
`BUILD_CONFIG' would be "dev" in either case, as far as I can tell

Reply:
TLDR: Ok, let's stick with "developer build" then.

I originally requested "developer build" or "nightly build" (i.e. use the one that is most appropriate for a given build), but you are correct that the config string doesn't make this distinction anymore. Nightly still appears [in the CI scripts](https://github.com/musescore/MuseScore/blob/e927e90b00066addd3bd63d2461e012377b23222/build/ci/tools/make_build_mode_env.sh#L47) but you won't have access to it here.

Reply:
Should it be translatable? The old term "Unstable Prerelease for Version: %1" was translatable.
It would be actually only translated after the 4.0 translation round starts, so around a Beta.

Reply:
There's no harm in translating the string, but I don't think it really matters. It doesn't appear in release builds, and anybody using a development build would have had to navigate through the [nightly builds page](https://ftp.osuosl.org/pub/musescore-nightlies/) or one of the compilation guides, all of which are only available in English.

Reply:
I though so too. So will leave it as is.

Reply:
What the heck... why not just do it the same way as in 3.x. Or at least similar:
![image](https://user-images.githubusercontent.com/1786669/116442246-8f4e7380-a852-11eb-84fc-e77a5746e091.png)
Let me know.

Reply:
Ah, just let's do it
<!--from: `return QString::fromStdString(configuration()->museScoreVersion()) + (mu::framework::Version::unstable() ? " development build" : "");`-->